content/articles/building-a-loader-using-animations-in-tailwind-css/index.md

```diff
@@ -0,0 +1,171 @@
+
+- `bg-white` has given it a white color,
+- `rounded-full` will create a circle, but will eventually change to a pill-like shape after we add the dots in it.
+- `justify-center` and `item-center` both center align the pill.
+
+This gives us the following result:
+
+![Step 1 Result](img1.png)
+
+Now let's add something inside the 'boring' circle above. As mentioned earlier, we are going to add three `div`s which will create three dots.
+
+```html
+<div class="bg-blue-600 p-2 w-4 h-4 rounded-full"></div>
+<div class="bg-green-600 p-2 w-4 h-4 rounded-full"></div>
+<div class="bg-red-600 p-2 w-4 h-4 rounded-full"></div>
+```
+
+Our loader will now look like this:
+
+![Result](img2.png)
+
+In the above snippet, the dots are coloured blue, green, and red respectively.
+
+The space in between the three dots is achieved by adding the class `space-x-2` in the parent div. They are also horizontally aligned using `flex` class in the same wrapper.
+
+And of course, the dots have the same width and height of 14px and a padding of 8px.
+
+### Adding animations
+
+Its animation time!
+
+For the animation part, we will use `animate-bounce` class from Tailwind CSS and a few custom CSS properties.
+
+Lets get to it!
+
+Add `animate-bounce` to the three lines of code above like this:
+
+```html
+<div class="bg-blue-600 p-2  w-4 h-4 rounded-full animate-bounce"></div>
+<div class="bg-green-600 p-2 w-4 h-4 rounded-full animate-bounce"></div>
+<div class="bg-red-600 p-2  w-4 h-4 rounded-full animate-bounce"></div>
+```
+
+Now all our coloured circles are bouncing! However, they are bouncing up and down at the same time. I think we should make it a little bit more fun 🤔. This can be achieved using animation properties in CSS.
+
+Let's give each circle a class name according to its color for faster reference.
+
+```html
+<div
+  class="bg-blue-600 p-2  w-4 h-4 rounded-full animate-bounce blue-circle"
+></div>
+<div
+  class="bg-green-600 p-2 w-4 h-4 rounded-full animate-bounce green-circle"
+></div>
+<div
+  class="bg-red-600 p-2  w-4 h-4 rounded-full animate-bounce red-circle"
+></div>
+```
+
+After the step above, we now write the class selectors inside a `<style>` tag in the head section. We will then add the `animation-delay` property for all selectors.
+
+Alternatively, you can create an external stylesheet file and link it to the html file.
+
+Our code should look like this:
+
+```css
+<style>
+	.blue-circle{
+		animation-delay: 0.1s;
+	}
+	.green-circle{
+		animation-delay: 0.2s;
+	}
+	.red-circle{
+		animation-delay: 0.3s;
+	}
+</style>
+```
+
+The above code has added time for which each circle would wait before it bounces. For instance, the red circle waits for 0.3 seconds before it bounces. It waits for the green circle to bounce before it does. The green circle on the other hand waits for the blue circle to bounce before it does
+
+Here's our very simple loader in just a few, simple steps!
+
+![Animated Loader](video.gif)
```

Comment:
@M0nicah 
I believe the referenced resource at this point is missing in the article folder. Could you please check on it.

cc: @kayere